lib/compat/wordpress-6.0/class-wp-theme-json-6-0.php

```diff
@@ -227,6 +227,7 @@ class WP_Theme_JSON_6_0 extends WP_Theme_JSON_5_9 {
 		'layout'          => array(
 			'contentSize' => null,
 			'wideSize'    => null,
+			'padding'     => null,
```

Comment:
I'm not sure why we're adding this when we can use the value at styles > spacing > padding.